build/golint.sh

```diff
@@ -22,7 +22,9 @@ TIMEOUT="10m"
 
 echo "Running golangci-lint..."
 
-golangci-lint run --timeout ${TIMEOUT} --skip-dirs ${SKIP_DIR_REGEX} -E maligned,whitespace,gocognit,unparam -e '`ctx` is unused'
+# https://github.com/mdempsky/maligned is deprecated.
+# maligned | The repository of the linter has been archived by the owner. Replaced by govet 'fieldalignment'.
```

Comment:
This comment is only helpful in the context of this change. It belongs in the PR description. It adds noise going forward here in the script, please remove.
```suggestion
```

Reply:
fixed